source/_integrations/amcrest.markdown

```diff
@@ -205,6 +206,129 @@ Service data attribute | Optional | Description
 -|-|-
 `entity_id` | no | Name(s) of entities, e.g., `camera.living_room_camera`.
 
+#### Service `ptz_control`
+
+If your Amcrest or Dahua camera supports PT/PTZ, you will be able to pan, tilt or zoom your camera. Zoom is only supported on PTZ
+cameras with optical zoom. It does not support zooming of Varifocal lenses.
+
+| Service data attribute | Description |
+| -----------------------| ----------- |
+| `entity_id` | String or list of strings that point at `entity_id`s of cameras. Use `entity_id: all` to target all. |
+| `movement` | Direction of the movement. Allowed values: `zoom_in`, `zoom_out`, `up`, `down`, `left`, `right`, `right_up`, `right_down`, `left_up`,  `left_down` |
+| `travel_time` | (Optional) Travel time in fractional seconds. Allowed values: `0` to `1`. Default: `0.2`. |
+
+### Example card with controls
+
+<p class='img'>
+  <img src='/images/integrations/amcrest/amcrest_ptz.jpg' alt='Screenshot using a picture-elements with PTZ controls.'>
```

Comment:
You need to add the picture file.

Reply:
I did the "Upload file" with a new "amcrest_ptz.jpg" image from the main page which I think was a mistake, but I don't see it listed to remove it.  
 
If I navigate to https://github.com/GaryOkie/home-assistant.github.io/tree/patch-1 , I can then navigate to /source/images/integrations .    There isn't a create folder option I could see, but I tried create file, hoping it would give me a folder option.  Nope, now I've got an empty amcrest file that needs to be removed.

Lastly, when reviewing the rendered markdown, why is the description for ptz_control not displaying?   I can't see anything wrong.

Reply:
First, the picture still isn't right. You have two files in the amcrest folder, one named .JPG and the other .jpg. I think you need to delete the .jpg one and rename the other from .JPG to .jpg, at the very least. I don't know if the `<img src=...>` link is right or not.

Regarding the ptz_control entity_id row, I think that's because you have:
```
| Service data attribute | Description |
| -----------------------| ----------- |
```
instead of:
```
Service data attribute | Optional | Description
-|-|-
```
Look at the other service descriptions as an example.

Reply:
> Still not sure about where that initial uploaded image went or if it will cause an issue.

Looks like you committed it to the `next` branch instead of your `patch-1` branch.

Reply:
thanks.  That's fixed now.